session/mock_bootstrap.go

```diff
@@ -140,13 +155,26 @@ func modifyBootstrapVersionForTest(store kv.Storage, ver int64) int64 {
 	return ver
 }
 
+const (
+	defaultMockUpgradeToVerLatest = 0
+	// MockSimpleUpgradeToVerLatest is used to indicate the use of the simple mock bootstrapVersion.
+	MockSimpleUpgradeToVerLatest = 1
```

Comment:
What does it mean 'Simple' here? The function 'mockSimpleUpgradeToVerLatest'?

And, the function and the enum are sharing the same name seems not a good idea.

Reply:
Comments were added. Another name has a good suggestion can be put forward, I change it